django/db/backends/base/schema.py

```diff
@@ -951,9 +951,7 @@ def _alter_field(
                 fks_dropped.add((old_field.column,))
                 self.execute(self._delete_fk_sql(model, fk_name))
         # Has unique been removed?
-        if old_field.unique and (
-            not new_field.unique or self._field_became_primary_key(old_field, new_field)
-        ):
+        if old_field._unique and not new_field._unique:
```

Comment:
We're still using `_unique` here

Reply:
I'm not clear on why we're not allowed to use the `_unique` property. Is it because it's supposed to be private on the `Field` class, so no one else is supposed to use it? If so, is there a public property that means the same thing, that we can use?

Reply:
Something like `if old_field.unique and not old_field.primary_key and not new_field.unique:` might work 🤔

Reply:
OK, I used that.